src/Service.Tests/Unittests/RestServiceUnitTests.cs

```diff
@@ -145,16 +145,10 @@ public static void InitializeTest(string restRoutePrefix, string entityName)
             queryManagerFactory.Setup(x => x.GetQueryBuilder(It.IsAny<DatabaseType>())).Returns(queryBuilder);
             queryManagerFactory.Setup(x => x.GetQueryExecutor(It.IsAny<DatabaseType>())).Returns(queryExecutor);
 
-            Mock<ISqlMetadataProvider> sqlMetadataProvider = new();
-            string outParam;
-            sqlMetadataProvider.Setup(x => x.TryGetEntityNameFromPath(It.IsAny<string>(), out outParam)).Returns(true);
-            Dictionary<string, string> _pathToEntityMock = new() { { entityName, entityName } };
-            sqlMetadataProvider.Setup(x => x.TryGetEntityNameFromPath(It.IsAny<string>(), out outParam))
-                               .Callback(new metaDataCallback((string entityPath, out string entity) => _ = _pathToEntityMock.TryGetValue(entityPath, out entity)))
-                               .Returns((string entityPath, out string entity) => _pathToEntityMock.TryGetValue(entityPath, out entity));
-
-            metadataProviderFactory.Setup(x => x.GetMetadataProvider(It.IsAny<string>())).Returns(sqlMetadataProvider.Object);
+            RuntimeConfig loadedConfig = provider.GetConfig();
+            loadedConfig.TryAddEntityPathNameToEntityName(entityName, entityName);
```

Comment:
This is because the metadataprovider was being used for the trygetentitynamefrom path function. since that logic is now in runtime config we no longer need it here.